src/Providers/BlamableServiceProvider.php

```diff
@@ -10,6 +10,12 @@ class BlamableServiceProvider extends ServiceProvider
 {
     public function boot()
     {
+        if ($this->app->runningInConsole()) {
+            $this->publishes([
+                ASTALPAERT_BLAMABLE_ROOT.'/config/astalpaert-blamable.php' => config_path('astalpaert-blamable.php'),
```

Comment:
You could use:

`__DIR__ . '/../config/blamable.php' => config_path('astalpaert-blamable.php'),`